README.md

```diff
@@ -4,7 +4,9 @@
 
 This contract contains a collection of deployments of the contract of the [Safe contracts repository](https://github.com/safe-global/safe-contracts). 
 
-For each deployment the address on the different networks and the abi files are available. To get an overview of the available versions check the available [json assets](./src/assets/).
+For each deployment the address on the different networks and the abi files are available. To get an overview of the available versions check the available [json assets](./src/assets/) and [Summary Table](./SUMMARY.md).
+
+To regenerate the summary table to include new deployments run: `ts-node src/generateSummary.ts` or `npx src/generateSummary.ts`.
```

Comment:
please add a script in the package.json for this.

Reply:
Done

Reply:
https://github.com/safe-global/safe-deployments/pull/228/commits/65be0d482d35a984599c4998492df2e5a1ce4204